public/static/js/patches.js

```diff
@@ -35,7 +35,7 @@ mciModule.controller('PatchesController', function($scope, $filter, $http, $wind
       $scope.patches = data['UIPatches'];
 
       _.each($scope.patches, function(patch) {
-          patch.canEdit = ($window.user.Id === patch.Author ) || $window.isSuperUser
+          patch.canEdit = (($window.user.Id === patch.Author ) || $window.isSuperUser) && patch.alias !== "__commit_queue"
```

Comment:
don't you want to look at if the requester is commitqueue, not if the alias was used? or can you use the CanEdit field that you setup in service/version.go

Reply:
Patches don't have requesters, and it doesn't look like CanEdit is available here since patches.html seems to be defined at a a different point

Reply:
(@john-m-liu could verify, I'm a little murky on how angular comes together)

Reply:
There isn't anything angular-specific here. If you wanted to though, you could check the requester on the patch's version

Reply:
We do use patch alias to determine if a patch is a commit queue item elsewhere also so I think it fits convention to just do that.